tox.ini

```diff
@@ -47,8 +47,9 @@ commands =
 [testenv:lint-mypy]
 deps =
     mypy
+    lxml
 commands =
-    mypy twine tests
```

Comment:
It looks like `tests` was added optimistically in the [initial work](https://github.com/pypa/twine/commit/92cc1c03116053b03173e22c2d6d8fa38a56c045#diff-b91f3d5bd63fcd17221b267e851608e8R43), with the caveat that any mypy errors were ignored. Since that's no longer the case, I think ignoring the tests (for now) will make it easier to get the source fully typed.